docs/integration-guides/wordpress.md

```diff
@@ -163,12 +143,10 @@ Congratulations! You have configured the Beyond Identity Admin Console and the O
 
 In this final step, you'll attempt to log in and verify successful authentication.
 
-1. Using a device where you have [created an identity and then enrolled a credential](#create-an-identity), go to http://your_hostname/wp-login.php to test authentication.
-
-2. Click the **Login with OpenID Connect** button. You'll be redirected to the Beyond Identity Web Authenticator.
+1. Go to http://your_hostname/wp-login.php to test authentication.
 
-4. You may see a step-up authentication prompt, depending on how **Policy** is set up for your tenant.
+1. Click the **Login with OpenID Connect** button. You'll be redirected to the Beyond Identity Web Authenticator.
 
-5. At the conclusion of a successful authentication, you'll be redirected to your WordPress admin console. 
+1. At the conclusion of a successful authentication, you'll be redirected to your WordPress admin console (or your users to your site depending on thier role). 
 
-6. From the Beyond Identity Admin Console, go to **Events** to view several successful authentication events. 
+1. From the Beyond Identity Admin Console, go to **Events** to view several successful authentication events. 
```

Comment:
Just one note of caution. I noticed that if some of a numbered list is part of an include file, and the numbered list continues, then the numbered list will start over at 1. The way I found around that is to continue the numbered list with the next number. I don't see any case of this in these files though.